{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/views/public.py

```diff
@@ -36,4 +38,6 @@ def get(self):
         # something like Cronitor is hitting this URL repeatedly to monitor uptime.
         log.info('ping-db ok')
 
+        ctasks.ping.apply_async()
```

Comment:
Can we please remove this from the CC?  You can add it to your project if you want and tell me how it goes, but I don't want to "bless" this yet.

Reply:
Sorry, NM, I didn't read this right.